makefiles/tools/kconfiglib.inc.mk

```diff
@@ -3,9 +3,13 @@ MENUCONFIG ?= $(RIOTTOOLS)/kconfiglib/riot_menuconfig.py
 BASE_MENUCONFIG ?= $(RIOTTOOLS)/kconfiglib/menuconfig.py
 GENCONFIG := $(RIOTTOOLS)/kconfiglib/genconfig.py
 
+ifeq ($(RIOT_CI_BUILD),1)
+  QUIETMESSAGES=":"
```

Comment:
What is `:` supposed to do?

Reply:
Apparently the make way to optionally quiet the echos...

Reply:
Very open to suggestions.

Reply:
huh TIL, I only knew about `@` to silence echoing the command